src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -15,20 +15,23 @@
 -->
 
 <kd-help-section>
-  <md-input-container class="md-block" md-is-error="ctrl.isNameError()">
+  <md-input-container class="kd-deploy-block" md-is-error="ctrl.isNameError()">
```

Comment:
@floreks: kd-deploy-block is used everywhere else in the form. The CSS for it alters the margins values. If it is currently ignored, wouldn't it be a better idea to keep it consistent with the rest of the form? Potential (however unlikely) alterations to the surroundings of this field will not require alterations to this field's code...?
Re. $baseline-grid: this makes sense for width, but not height? @bryk made the same comment, but I don't have a final answer on this...